### ISSUE TITLE
Why are you using an alpha version of sass?

### ISSUE THREAD

**Issue by sgehrman** [mirisuzanne/compass-animation#10]
I was having a problem with middleman and found out the problem was that your animation gem was including an alpha version of sass.

== The Middleman is loading
== The Middleman is standing watch at http://0.0.0.0:4567
Warning. Error encountered while saving cache ./.sass-cache/6ae58e87ce79f79f84bd681b2fbed147db898c2b/screen.css.scssc: can't dump anonymous class #Class:0x007fd438fcba48
Warning. Error encountered while saving cache ./.sass-cache/6ae58e87ce79f79f84bd681b2fbed147db898c2b/_variables.css.scssc: can't dump anonymous class #Class:0x007fd438fcba48
Warning. Error encountered while saving cache ./.sass-cache/6ae58e87ce79f79f84bd681b2fbed147db898c2b/_fonts.css.scssc: can't dump anonymous class #Class:0x007fd438fcba48

Any reason why you are using that build?

**Comment by mirisuzanne** [comment]
Because this is an old plugin, and I haven't been maintaining it. This has all moved into the Compass master branch. I'll give it an update here, when I get a chance.

**Comment by stereoscott** [comment]
I began seeing sass cache errors after including the animation plugin, too. "Warning. Error encountered while saving cache" ... "can't dump anonymous class". I was trying to avoid referencing github gems (due to the way Amazon Elastic Beanstalk does their deployments), so I'm using the standard compass-rails 1.0.3 gem and the compass 0.12.2 gem. 

Thanks for the note about this moving into the Compass master branch. In the meantime, I've exported the necessary animation files into my vendor/assets (in a Rails app) and that will hold me over. 

Thanks for the great mixins and for sharing them with the community.
